CMakeLists.txt

```diff
@@ -123,7 +123,6 @@ if(${ARBORX_ENABLE_TESTS})
   endforeach()
   foreach(_file ${ArborX_HEADERS})
     foreach(_macro ${ArborX_MACROS})
-      file(STRINGS src/${_file} _includes_mpi REGEX "mpi.h")
```

Comment:
Why did you decide to make this change instead of tracking the place where the issue originated?

Reply:
I don't necessarily see something wrong with including headers that include the distributed implementation even if ArborX has been configured without MPI support.
I would also argue that not all the files that implement the distributed implementation include `mpi.h` directly (which was the reason for the previous commit to fail).